packages/quantic/cypress/page-objects/example-search.ts

```diff
@@ -0,0 +1,75 @@
+export const setupAliases = () =>
+  cy
+    .intercept(
+      'POST',
+      'https://platform.cloud.coveo.com/rest/search/v2?organizationId=searchuisamples'
+    )
+    .as('search')
```

Comment:
To be able to retrieve the aliases use them in the tests would it be better to have these "search", "result", "pager" in an exported object?

Reply:
I did try that, but the end result was not as good as I expected. The annoying thing is that you have to have to use the `@` prefix when referencing an alias.

So, using variables was ok in the page object file, but it became less readable in the test suite because of the translation function you have to call to prefix the alias.

In the end, using a plain string was just simpler.